docs/sources/mimir/get-started/_index.md

```diff
@@ -156,31 +156,30 @@ scrape_configs:
       - targets: ["localhost:9090"]
 ```
 
-## Configure Grafana Agent to write to Grafana Mimir
+## Configure Grafana Alloy to write to Grafana Mimir
 
-Add the following YAML snippet to one of your Agent metrics configurations (`metrics.configs`) in your Agent configuration file, and restart Grafana Agent:
+Use the `prometheus.remote_write` component in Grafana Alloy to send metrics to Grafana Mimir. For example:
 
-```yaml
-remote_write:
-  - url: http://localhost:9009/api/v1/push
 ```
```

Comment:
Let's verify this configuration, please.

Reply:
It's correct and matches the docs for that component. Maybe we should link to the component page though so users can see extra config options.